.drone.star

```diff
@@ -584,10 +584,10 @@ def uploadGuiTestLogs():
         },
         "environment": {
             "AWS_ACCESS_KEY_ID": {
-                "from_secret": "cache_s3_access_key",
+                "from_secret": "cache_public_s3_access_key",
             },
             "AWS_SECRET_ACCESS_KEY": {
-                "from_secret": "cache_s3_secret_key",
+                "from_secret": "cache_public_s3_secret_key",
```

Comment:
do we anctually need those at all?

Reply:
yes, otherwise, upload will fail like mentioned here https://github.com/owncloud/client/pull/9470#issuecomment-1049591535